virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -7,72 +7,103 @@ include::_attributes/common-attributes.adoc[]
-:!FeatureName:
-
-endif::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
-
-// Hiding in ROSA/OSD as FIPS not supported
-ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
-.FIPS mode
-
-If you install your cluster in xref:../../installing/overview/installing-fips.adoc#installing-fips-mode_installing-fips[FIPS mode], no additional setup is required for {VirtProductName}.
-endif::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
-
-[id="supported-platforms_preparing-cluster-for-virt"]
-== Supported platforms
+[id="compatible-platforms_{context}"]
+== Compatible platforms
 
 You can use the following platforms with {VirtProductName}:
 
 // Hiding in ROSA/OSD, as bare metal not supported
 ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
 * On-premise bare metal servers. See xref:../../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#virt-planning-bare-metal-cluster-for-ocp-virt_preparing-to-install-on-bare-metal[Planning a bare metal cluster for {VirtProductName}].
 
-* Amazon Web Services bare metal instances. See xref:../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations].
-//See link:https://access.redhat.com/articles/6409731[Deploy {VirtProductName} on AWS metal instance types]. // seems outdated with references to OCS - remove?
+//* {ibm-cloud-name} Bare Metal Servers. See link:https://access.redhat.com/articles/6738731[Deploy {VirtProductName} on {ibm-cloud-name} Bare Metal nodes].
+//+
+//--
+//ifdef::openshift-enterprise[]
+//:FeatureName: Installing OpenShift Virtualization on {ibm-cloud-name} Bare Metal Servers
+//include::snippets/technology-preview.adoc[]
+//:!FeatureName:
+//endif::[]
+//--
 
-* {ibm-cloud-name} Bare Metal Servers. See link:https://access.redhat.com/articles/6738731[Deploy {VirtProductName} on {ibm-cloud-name} Bare Metal nodes].
+* {ibm-z-name} or {ibm-linuxone-name} (s390x architecture) systems where an {product-title} cluster is installed in logical partitions (LPARs). See xref:../../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z_preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].
 +
 --
 ifdef::openshift-enterprise[]
-:FeatureName: Installing OpenShift Virtualization on {ibm-cloud-name} Bare Metal Servers
+:FeatureName: Using {VirtProductName} in a cluster deployed on s390x architecture
 include::snippets/technology-preview.adoc[]
 :!FeatureName:
 endif::[]
 --
 
-* {ibm-z-name} or {ibm-linuxone-name} (s390x architecture) systems where an {product-title} cluster is installed in logical partitions (LPARs). See xref:../../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z_preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].
+endif::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
```

Comment:
Is there an `ifdef` or `ifndef` for this `endif`?

Reply:
Good point! It was there, but I think it needs to be moved up to encompass the entire section.